File: pulumi/examples/python/digitalocean_loadbalancer_example.py
"""
Pulumi DigitalOcean Load Balancer Example (Python)

Setup:
  pip install pulumi pulumi_digitalocean
Usage:
  pulumi up
"""
import pulumi
import pulumi_digitalocean as do

lb = do.LoadBalancer('example-lb',
    region='nyc3',
  forwarding_rules=[{
    "entry_protocol": "http",
    "entry_port": 80,
    "target_protocol": "http",
    "target_port": 80
  }]
)
pulumi.export("load_balancer_id", lb.id)

File: pulumi-infra-project/pulumi/examples/typescript/digitalocean_loadbalancer_example.ts
// Pulumi DigitalOcean Load Balancer Example (TypeScript)
import * as pulumi from "@pulumi/pulumi";
import * as digitalocean from "@pulumi/digitalocean";

const lb = new digitalocean.LoadBalancer("example-lb", {
    region: "nyc3",
    forwardingRules: [{
        entryProtocol: "http",
        entryPort: 80,
        targetProtocol: "http",
        targetPort: 80
    }]
});

export const loadBalancerId = lb.id;
